package.json

```diff
@@ -27,7 +27,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "serve": "cd api && json-server -p 5002 -w dtty.json"
```

Comment:
Instead of having someone who wants to run your code do some extra steps, make it as easy as possible for them by making little scripts like this so they don't have to change directories.... do it for them!   (the `&&` allows you to chain shell commands)